### ISSUE TITLE
Remove internal address for OGC-Proxy in Prod

### ISSUE THREAD

**Issue by gjn** [geoadmin/mf-geoadmin3#4685]
For production, we are using `service-proxy.prod.bgdi.ch` as address of our ogc proxy.

We should create an official address for it: `proxy.geo.admin.ch` (even though we don't officially communicate it as service - just as print)

See https://github.com/geoadmin/mf-geoadmin3/blob/master/Makefile#L63
See https://github.com/geoadmin/mf-geoadmin3/blob/master/rc_prod#L11

As principle: never use technical addresses in production that is visible to the outside world. Hell awaits on the other end.

**Comment by procrastinatio** [comment]
No, everything should be done to discourage its use. The address should look ugly, random, and ideally, change from time to time.

**Comment by danduk82** [comment]
we can discuss this next week at weekly meeting. Anyway we should remove the addresses in `bgdi.ch`.

**Comment by danduk82** [comment]
won't do for tonight, change milestone.

**Comment by danduk82** [comment]
@gin we should discuss this again

**Comment by procrastinatio** [comment]
Decision?
